met-web/src/components/userManagement/listing/ActionsDropDown.tsx

```diff
@@ -18,21 +18,21 @@ export const ActionsDropDown = ({ selectedUser }: { selectedUser: User }) => {
     const { roles, userDetail } = useAppSelector((state) => state.user);
 
     const hasNoRole = (): boolean => {
-        if (selectedUser.main_group) {
+        if (selectedUser.main_role) {
             return false;
         }
         return true;
     };
 
     const isAdmin = (): boolean => {
-        if (selectedUser?.main_group == USER_GROUP.ADMIN.label) {
+        if (selectedUser?.main_role == USER_GROUP.ADMIN.label) {
```

Comment:
Are we able to remove "groups" here too? Maybe we could call this `USER_COMPOSITE_ROLE` OR `USER_ROLE` if it's not too confusing.

```suggestion
        if (selectedUser?.main_role == USER_ROLE.ADMIN.label) {
```

Reply:
As suggested considered USER_COMPOSITE_ROLE and updated everywhere